community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    alm-examples: |-
+      [{
+       "apiVersion": "hazelcast.com/v1alpha1",
+       "kind": "Hazelcast",
+       "metadata": {
+          "name": "hz"
+       },
+       "spec": {
+          "image": {
+            "tag": "3.11.2"
+          },
+          "hazelcast": {
+            "licenseKeySecretName": "hz-license-key"
+          },
+          "securityContext": {
+            "runAsUser": 65534,
+            "fsGroup": 65534
+          }
+       }
+      }]
+    categories: "Database"
+    certified: false
+    containerImage: hazelcast/hazelcast-enterprise-operator:0.0.1
+    createdAt: 2019-03-14T09:58:52Z
+    description: Install Hazelcast Enterprise cluster.
+    support: Hazelcast, Inc
+  name: hazelcast-operator.v0.0.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Hazelcast Enterprise cluster.
+      displayName: Hazelcast Enterprise
+      group: hazelcast.com
+      kind: Hazelcast
+      name: hazelcasts.hazelcast.com
```

Comment:
that is a good question. It was actually generated this way. They probably just put plural form of the "kind" name. The other option I've seen in other operators is "hazelcastclusters.hazelcast.com".

Reply:
`hazelcastclusters.hazelcast.com`... I liked that one.

Reply:
👍

Reply:
hmm, it means that everything will have to be called: `HazelcastCluster` instead of `Hazelcast`. Then, you will list your hazelcast cluster like: 
`kubectl get HazelcastCluster my-hz` instead of `kubectl get Hazelcast my-hz`  and that the user definition of the cluster will be like: 
```
apiVersion: hazelcast.com/v1alpha1
kind: HazelcastCluster
metadata:
  name: hz
spec:
  image:
    tag: "3.11.2"
  hazelcast:
    licenseKeySecretName: "hz-license-key"
```

I personally like `Hazelcast` more, because it's clear and to the point. One may argue that `HazelcastCluster` is closer to the truth, because you actually deploy the hazelcast cluster, however, the truth truth is that you really deploy not just the cluster, but the whole Hazelcast Ecosystem (with Management Center, etc.).

Reply:
I agree. Lets keep hazelcasts...